ddp-workspace/projects/ddp-testboston/src/app/components/footer/footer.component.html

```diff
@@ -29,7 +29,8 @@
                     </a>
                 </li>
                 <li class="navigation__item">
-                    <a class="footer-link" routerLink="/privacy-policy" routerLinkActive="footer-link_active" translate>
+                    <a class="footer-link" [routerLink]="'/' + appRoutes.Privacy" routerLinkActive="footer-link_active"
+                        translate>
```

Comment:
The privacy link is a  new page we have done yet, right?

Reply:
Right.